.eslintrc.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
     ecmaVersion: 2015,
     sourceType: "module"
   },
-  parser: '@typescript-eslint/parser',
+  // parser: '@typescript-eslint/parser',
```

Comment:
why this change?

Reply:
It goes back and forth. :)  The uncomment never should have gotten committed.

Reply:
Do you know if you can use `parser` only for specific paths?

Reply:
you can! via the overrides options (which I recommend moving to _only_ overrides, as I've done here: https://github.com/NullVoxPopuli/eslint-configs)

Reply:
Nice!  For some reason I was thinking that something as high level as the parser couldn't be configured in overrides. Seems to work just fine though.